systemd/cloud-init-local.service.tmpl

```diff
@@ -1,23 +1,45 @@
 ## template:jinja
 [Unit]
 Description=Initial cloud-init job (pre-networking)
-{% if variant in ["ubuntu", "unknown", "debian"] %}
+{% if variant in ["ubuntu", "unknown", "debian", "rhel" ] %}
 DefaultDependencies=no
 {% endif %}
 Wants=network-pre.target
+{% if variant not in ["rhel"] %}
 After=hv_kvp_daemon.service
```

Comment:
Are there specific needs where RHEL doesn't want to log failures/warnings Azure's KVP backplane? Or is the desire to avoid this being present on non-Azure platforms?

Reply:
Nope, as above I was just not sure. If you tell me that After=X does not have an impact if the service is not installed, then I am happy to leave this for rhel too :)

Reply:
Yes confirmed. We ship stock After=hv_kvp_deaemon.sevice definitions on other distros only impact systemd ordering if hv_kvp_daemon.service is present on the system (which should be Azure cloud images)